steps_test.go

```diff
@@ -116,7 +116,7 @@ func (s *StepTestSuite) TestSimpleConfig() {
 	s.writeSimpleConfig()
 
 	for _, step := range syncSteps {
-		step.fn(s.c, s.c)
+		s.Require().NoError(step.fn(s.c, s.c))
```

Comment:
I think these are failing in CI on the success case where we return `errSkipped`.

Reply:
:+1: thanks! I added a helper function, `RequireNilOrSkipped` for these `step.fn` calls, which will hopefully fix these tests.